games/minimal/mods/testformspec/formspec.lua

```diff
@@ -0,0 +1,381 @@
+	style[one_btn14:hovered;bgimg=testformspec_bg_hovered.png;fgimg=testformspec_hovered.png;textcolor=yellow]
+	style[one_btn14:pressed;bgimg=testformspec_bg_pressed.png;fgimg=testformspec_pressed.png;textcolor=blue]
+	style[one_btn14:hovered+pressed;textcolor=purple]
+	image_button[0,9.6;1,1;testformspec_button_image.png;one_btn14;Bg]
+
+	style[one_btn15;border=false;bgimg=testformspec_bg.png;bgimg_hovered=testformspec_bg_hovered.png;bgimg_pressed=testformspec_bg_pressed.png]
+	item_image_button[1.25,9.6;1,1;testformspec:item;one_btn15;Bg]
+
+	style[one_btn16;border=false;bgimg=testformspec_bg_9slice.png;bgimg_hovered=testformspec_bg_9slice_hovered.png;bgimg_pressed=testformspec_bg_9slice_pressed.png;bgimg_middle=4,6]
+	button[2.5,9.6;2,1;one_btn16;9-Slice Bg]
+
+
+
+	container[2.75,0]
+
+	style[one_tb1;textcolor=Yellow]
+	tabheader[0,3;2.5,0.63;one_tb1;Yellow,Text,Tabs;1;false;false]
+
+	style[one_f1;textcolor=yellow]
+	field[0,4.25;2.5,0.8;one_f1;Field One;Yellow Text]
+
+	style[one_f2;border=false;textcolor=cyan]
+	field[0,5.75;2.5,0.8;one_f2;Field Two;Borderless Cyan Text]
+
+	style[one_f3;textcolor=yellow]
+	textarea[0,7.025;2.5,0.8;one_f3;Label;]] ..
+		minetest.formspec_escape("Yellow Text\nLine two") .. [[ ]
+
+	style[one_f4;border=false;textcolor=cyan]
+	textarea[0,8.324999999999999;2.5,0.8;one_f4;Label;]] ..
+		minetest.formspec_escape("Borderless Cyan Text\nLine two") .. [[ ]
+
+	container_end[]
+]]
+
+local scroll_fs =
+	"button[8.5,1;4,1;outside;Outside of container]"..
+	"box[1,1;8,6;#00aa]"..
+	"scroll_container[1,1;8,6;scrbar;vertical]"..
+		"button[0,1;1,1;lorem;Lorem]"..
+		"button[0,10;1,1;ipsum;Ipsum]"..
+		"pwdfield[2,2;1,1;lorem2;Lorem]"..
+		"list[current_player;main;4,4;1,5;]"..
+		"box[2,5;3,2;#ffff00]"..
+		"image[1,10;3,2;testformspec_item.png]"..
+		"image[3,1;testformspec_item.png]"..
+		"item_image[2,6;3,2;testformspec:node]"..
+		"label[2,15;bla Bli\nfoo bar]"..
+		"item_image_button[2,3;1,1;testformspec:node;itemimagebutton;ItemImageButton]"..
+		"tooltip[0,11;3,2;Buz;#f00;#000]"..
+		"box[0,11;3,2;#00ff00]"..
+		"hypertext[3,13;3,3;;" .. hypertext_basic .. "]" ..
+		"container[0,18]"..
+			"box[1,2;3,2;#0a0a]"..
+			"scroll_container[1,2;3,2;scrbar2;horizontal;0.06]"..
+				"button[0,0;6,1;butnest;Nest]"..
+				"label[10,0.5;nest]"..
+			"scroll_container_end[]"..
+			"scrollbar[1,0;3.5,0.3;horizontal;scrbar2;0]"..
+		"container_end[]"..
+		"dropdown[0,6;2;hmdrpdwn;apple,bulb;1]"..
+		"image_button[0,4;2,2;testformspec_button_image.png;imagebutton;bbbbtt;false;true;testformspec_pressed.png]"..
+		"box[1,22.5;4,1;#a00a]"..
+	"scroll_container_end[]"..
+	"scrollbaroptions[max=170]".. -- lowest seen pos is: 0.1*170+6=23 (factor*max+height)
+	"scrollbar[7.5,0;0.3,4;vertical;scrbar;0]"..
+	"scrollbar[8,0;0.3,4;vertical;scrbarhmmm;0]"..
+	"dropdown[0,6;2;hmdrpdwnnn;Outside,of,container;1]"
+
+--style_type[label;textcolor=green]

```

Comment:
Outcommented code should either be removed or made optional using configuration variables...